lib/config/configuration.go

```diff
@@ -992,6 +1000,78 @@ func applyAppsConfig(fc *FileConfig, cfg *service.Config) error {
 	return nil
 }
 
+// applyMetricsConfig applies file configuration for the "metrics_service" section.
+func applyMetricsConfig(fc *FileConfig, cfg *service.Config) error {
+	// Metrics is enabled.
+	cfg.Metrics.Enabled = true
+
+	if fc.Metrics.ListenAddress != "" {
+		addr, err := utils.ParseHostPortAddr(fc.Metrics.ListenAddress, int(defaults.MetricsListenPort))
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		cfg.Metrics.ListenAddr = addr
+	}
```

Comment:
please remove the outer condition
`cfg.Metrics.ListenAddr` should be populated and `utils.ParseHostPortAddr` will handle defaulting when empty